src/gc.rs

```diff
@@ -0,0 +1,257 @@
+use std::{
+    alloc::{AllocRef, Layout},
+    any::Any,
+    ffi::c_void,
+    fmt,
+    marker::{PhantomData, Unsize},
+    mem::{forget, transmute, ManuallyDrop, MaybeUninit},
+    num::NonZeroUsize,
+    ops::{CoerceUnsized, Deref, DerefMut},
+    ptr::NonNull,
+};
+
+use crate::boehm;
+use crate::GC_ALLOCATOR;
+
+/// A garbage collected pointer. 'Gc' stands for 'Garbage collected'.
```

Comment:
I think the second sentence is a bit redundant :)

Reply:
Yeah, fixed here: d71cc8fb0c2e50977c78b7f6a713dcc8ec7471da